lib/l10n/app_en.arb

```diff
@@ -1,51 +1,202 @@
 {
   "@@locale": "en",
 
-  "price": "{amount}€",
-  "@price": {
-      "placeholders": {
-          "amount": {
-              "example": "123.45",
-              "description": "amount"
-          }
-      }
+  "cancel": "Cancel",
+  "more": "More",
+  "yes": "Yes",
+  "no": "No",
+  "failureSnackBar": "Something went wrong.",
+
+  "riders": "Riders",
+  "seats": "Seats",
+  "seatsCount": "{count, plural, =1{{count} seat}, other{{count} seats}}",
+  "@seatsCount": {
+    "placeholders": {
+      "count": {}
+    }
   },
-  "cancel": "cancel",
-  "confirm": "confirm",
-  "login": "Login",
-  "register": "Register",
+
+  "authEmailLinkInvalid": "Email link is invalid or has expired.",
+
+  "formEmail": "Email",
+  "formEmailHint": "Enter your email address",
+  "formEmailValidateEmpty": "Please enter your email address",
+  "formEmailValidateInvalid": "Please enter a valid email address",
+  "formPassword": "Password",
+  "formPasswordHint": "Enter your password",
+  "formPasswordValidateEmpty": "Please enter your password",
+  "formPasswordValidateMinLength": "Password must be at least 8 characters",
+  "formPasswordValidateNumber": "Password must contain at least one number",
+  "formPasswordValidateUppercase": "Password must contain at least one uppercase letter",
+  "formPasswordValidateLowercase": "Password must contain at least one lowercase letter",
+  "formPasswordValidateSpecial": "Password must contain at least one special character",
+  "formPasswordConfirm": "Confirm Password",
+  "formPasswordConfirmHint": "Re-enter your password",
+  "formPasswordConfirmValidateEmpty": "Please confirm your password",
+  "formPasswordConfirmValidateMatch": "Passwords do not match",
+  "formDate": "Date",
+  "formTime": "Time",
+  "formTimeValidateEmpty": "Please enter a time",
+  "formTimeValidateFuture": "Please enter a time in the future",
```

Comment:
Find das Wort "choose" schöner da man bei der Zeit und beim Datum an sich ja eigentlich nur auswählt.
"Please choose a time"
"in the future" ist zwar die genaue Übersetzung bin mir aber noch unsicher ob sich das gut anhört.
eventuell: "Please choose a upcoming Time" oder "Please choose a future time"
aber an sich Sieht alles gut aus.

Reply:
Hm ich finde "choose" auch schöner an der Stelle (hat natürlich eigentlich nichts mit der Pull Request zu tun, aber ich mache es trotzdem mal dazu).

zu "in the future" finde ich deine Vorschläge nicht besser, aber ich muss mal überlegen ob mir noch was einfällt

(Sidenote: Du solltest vielleicht lieber die deutschen Übersetzungen durchgehen, an den englischen Strings habe ich ja einfach gar nichts verändert)